src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -28,7 +28,7 @@ export type BreadcrumbsProps = {
     items: Breadcrumb[];
 };
 
-export const Breadcrumbs: FC<BreadcrumbsProps> = ({ items }) => {
```

Comment:
When removing `FC<Props>`, don't forget to set the return type of the function to `ReactElement` otherwise we might return boolean and it won't complain

Reply:
Interesting! I guess I have to add the return type in every component `FC` have been removed. Might need to update the Fondue docs as well. And the template when running `npm run component:create`.

Reply:
Done! Added in some components and I created a task in clickup to remove `FC` in remaining components still using it. So this PR doesn't get too big.